tests/unit/containers/tutor-profile/security-block/SecurityBlock.spec.jsx

```diff
@@ -0,0 +1,92 @@
+import {
+  screen,
+  render,
+  fireEvent,
+  waitForElementToBeRemoved
+} from '@testing-library/react'
+import SecurityBlock from '~/containers/tutor-profile/security-block/SecurityBlock'
+
+const changeInputValue = (label, value) => {
+  fireEvent.change(screen.getByLabelText(label), { target: { value } })
+}
+
+describe('SecurityBlock', () => {
+  beforeEach(() => {
+    render(<SecurityBlock />)
+  })
+
+  it('renders title and description', () => {
+    expect(
+      screen.getByText('editTutor.passwordSecurityTab.title')
```

Comment:
according to our code convension, move screen.getByText('editTutor.passwordSecurityTab.title') to a variable

Reply:
Thank you for conversation! refactored